src/NPlaylist/IPlaylist.cs

```diff
@@ -0,0 +1,17 @@
+using System.Collections.Generic;
+
+namespace NPlaylist
+{
+    public interface IPlaylist
+    {
+        IDictionary<string, string> Tags { get; }
+        IList<IPlaylistItem> PlaylistItems { get; }
+    }
+
+    public interface IPlaylist<T> : IPlaylist where T : IPlaylistItem
+    {
```

Comment:
Add the concrete items property

```suggestion
{
    IEnumerable<T> Items { get; }
```